lib/matplotlib/_constrained_layout.py

```diff
@@ -84,10 +85,13 @@ def do_constrained_layout(fig, h_pad, w_pad,
        A value of 0.2 for a three-column layout would have a space
        of 0.1 of the figure width between each column.
        If h/wspace < h/w_pad, then the pads are used instead.
-
```

Comment:
For consistency with above, this empty line should be here (and one added after the `rect` description) or all above also removed.

Reply:
Sure, though this is private, so never rendered in the docs ;-)